dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -9618,6 +9618,14 @@ components:
     LambdaEvent:
       type: object
       properties:
+        entryNames:
```

Comment:
A little curious that the two descriptions are the same. What happens if you try `@ArraySchema`?
https://stackoverflow.com/questions/60002234/how-to-annotate-array-of-objects-response-in-swagger

Reply:
No longer applicable after newest PR changes